erdblick_app/app/view.service.ts

```diff
@@ -0,0 +1,33 @@
+import {Injectable} from "@angular/core";
+import {BehaviorSubject} from "rxjs";
+import {Cartesian3} from "cesium";
+
+@Injectable({providedIn: 'root'})
+export class ViewService {
```

Comment:
Please add a comment here, explaining why this service is needed. Something like

/**
 * The view service is required to buffer Cesium view settings, which may be set via the 
 * URL, before the actual Cesium view is available.
 */ 

I wonder though, why can't this be merged with ParameterService?